### PR TITLE
Added log messages w/ the version number.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: CromIAM/src/main/scala/cromiam/server/CromIamServer.scala [hunk 1]
```diff
@@ -25,6 +26,12 @@ object CromIamServer extends HttpApp with CromIamApiService with SwaggerService
   override val oauthConfig: SwaggerOauthConfig = configuration.swaggerOauthConfig
 
   def run(): Unit = {
+    log.info(s"Version {}", VersionUtil.getVersion("cromiam"))
+    /*
+    TODO: We're not passing in our actor system so a new one is getting created by akka.http.scaladsl.server.HttpApp.
```
Comment (cjllanwarne) [inline CromIAM/src/main/scala/cromiam/server/CromIamServer.scala]: ToL: TODOs are probably not the place to put "let's circle round to this at some point" issues.

Can we github it?

File: engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala [hunk 2]
```diff
@@ -50,6 +51,7 @@ class SingleWorkflowRunnerActor(source: WorkflowSourceFilesCollection,
   override lazy val jobStoreActor = context.actorOf(EmptyJobStoreActor.props, "JobStoreActor")
   override lazy val subWorkflowStoreActor = context.actorOf(EmptySubWorkflowStoreActor.props, "SubWorkflowStoreActor")
 
+  log.info("{}: Version {}", Tag, VersionUtil.getVersion("cromwell-engine"))
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala]: What's `Tag` and why doesn't the server mode get one?

Comment (kshakir) [reply engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala]: https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L258

Logging for Single vs. Server is a complete mystery to me. This syntax was just copied from other existing logging. Ex:

https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L57

https://github.com/broadinstitute/cromwell/blob/63637a61d138f87de99f4129f8b25a5a18d8088d/engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala#L181

File: CromIAM/src/main/scala/cromiam/server/CromIamServer.scala (hunk 1 above)
Comment (kshakir) [reply CromIAM/src/main/scala/cromiam/server/CromIamServer.scala]: Filed and linked: https://github.com/broadinstitute/cromwell/issues/3851

File: engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala (hunk 2 above)
Comment (cjllanwarne) [reply engine/src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala]: 🤷‍♂️

**Pull request merged**